src/server/cluster/cluster_family.cc

```diff
@@ -718,7 +718,6 @@ void ClusterFamily::DflySlotMigrationStatus(CmdArgList args, ConnectionContext*
   if (reply.empty()) {
     rb->SendSimpleString(StateToStr(MigrationState::C_NO_STATE));
   } else if (!node_id.empty()) {
-    DCHECK_EQ(reply.size(), 1UL);
     rb->SendSimpleString(reply[0]);
   } else {
     rb->SendStringArr(reply);
```

Comment:
Perhaps always do `rb->SendStringArr(reply);`, even if `node.empty() == true`, because this would allow callers to reduce ambiguity..